right_gimbal:
  extrinsic_T: [0.5, 0, 0 ]

left_gimbal:
  extrinsic_T: [-0.5, 0, 0 ]
